src/main/java/com/minecolonies/coremod/colony/TriggerColonyAchievements.java

```diff
@@ -6,28 +6,28 @@
 /**
  * Trigger the corresponding colony achievement.
  */
-public final class TriggerColonyAchievements
-{
-    private static final String TAG_LUMBERJACK_SAPLINGS   = "saplings";
-    private static final String TAG_LUMBERJACK_TREES      = "trees";
-    private static final String TAG_FISHERMAN_FISH        = "fish";
-    private static final String TAG_BUILDER_HUTS          = "huts";
-    private static final String TAG_GUARD_MOBS            = "mobs";
-    private static final String TAG_FARMER_CARROTS        = "carrots";
-    private static final String TAG_FARMER_POTATOES       = "potatoes";
-    private static final String TAG_FARMER_WHEAT          = "wheat";
-    private static final String TAG_MINER_DIAMONDS        = "diamonds";
-    private static final String TAG_MINER_ORES            = "ores";
+public final class TriggerColonyAchievements {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)